### ISSUE TITLE
You can get a module error because persismOriginalValue is serializable to json or xml

### ISSUE THREAD

**Issue by sproket** [sproket/Persism#38]
If you try serializing to xml you can't extend from PersistableObject since it has a field which is not accessible to the modules.

```
[GameManager] Failed making field 'net.sf.persism.PersistableObject#persismOriginalValue' accessible; either increase its visibility or write a custom TypeAdapter for its declaring type.
com.google.gson.JsonIOException: Failed making field 'net.sf.persism.PersistableObject#persismOriginalValue' accessible; either increase its visibility or write a custom TypeAdapter for its declaring type.
	at com.google.gson@2.10/com.google.gson.internal.reflect.ReflectionHelper.makeAccessible(ReflectionHelper.java:38)
	at com.google.gson@2.10/com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:287)
	at com.google.gson@2.10/com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:130)
	at com.google.gson@2.10/com.google.gson.Gson.getAdapter(Gson.java:546)
	at com.google.gson@2.10/com.google.gson.internal.bind.CollectionTypeAdapterFactory.create(CollectionTypeAdapterFactory.java:53)
	at com.google.gson@2.10/com.google.gson.Gson.getAdapter(Gson.java:546)
	at com.google.gson@2.10/com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.createBoundField(ReflectiveTypeAdapterFactory.java:160)
	at com.google.gson@2.10/com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:295)
	at com.google.gson@2.10/com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:130)
	at com.google.gson@2.10/com.google.gson.Gson.getAdapter(Gson.java:546)
	at com.google.gson@2.10/com.google.gson.Gson.toJson(Gson.java:817)
	at com.google.gson@2.10/com.google.gson.Gson.toJson(Gson.java:795)
	at com.google.gson@2.10/com.google.gson.Gson.toJson(Gson.java:742)
	at com.google.gson@2.10/com.google.gson.Gson.toJson(Gson.java:719)
	at game.goldchest/game.ui.GameManager.saveGame(GameManager.java:457)
	at game.goldchest/game.ui.menu.CampMenu.lambda$new$0(CampMenu.java:46)
	at game.goldchest/game.ui.MenuPanel.doSelect(MenuPanel.java:359)
	at game.goldchest/game.ui.MenuPanel.menuKeyEvent(MenuPanel.java:273)
	at game.goldchest/game.ui.MenuPanel.keyPressed(MenuPanel.java:202)
	at game.goldchest/game.ui.KeyDispatcher.dispatchKeyEvent(KeyDispatcher.java:64)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.preDispatchKeyEvent(DefaultKeyboardFocusManager.java:1144)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(DefaultKeyboardFocusManager.java:1020)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:848)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4882)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2780)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4833)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:746)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:744)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:743)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private java.lang.Object net.sf.persism.PersistableObject.persismOriginalValue accessible: module sproket.github.io.persism does not "opens net.sf.persism" to module com.google.gson
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
	at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:178)
	at java.base/java.lang.reflect.Field.setAccessible(Field.java:172)
	at com.google.gson@2.10/com.google.gson.internal.reflect.ReflectionHelper.makeAccessible(ReflectionHelper.java:35)
	... 43 common frames omitted


```

**Comment by sproket** [comment]
This is gson specific.

**Comment by sproket** [comment]
This is fixed with 2.2.1 to be released soon.
